build.sbt

```diff
@@ -33,28 +32,27 @@ lazy val microservice = Project(appName, file("."))
 
     ScoverageKeys.coverageMinimumStmtTotal := 90,
     ScoverageKeys.coverageMinimumBranchTotal := 90,
-    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageFailOnMinimum := false,
     ScoverageKeys.coverageHighlighting := true,
     TwirlKeys.templateImports ++= Seq(
       "config.AppConfig",
       "uk.gov.hmrc.govukfrontend.views.html.components._",
       "uk.gov.hmrc.hmrcfrontend.views.html.components._"
     ),
 
-    scalacOptions ++= Seq(
-      "-P:silencer:pathFilters=routes",
-      "-P:silencer:pathFilters=target/.*",
-      "-Wunused:imports", "-Wunused:params",
-      "-Wunused:patvars", "-Wunused:implicits",
-      "-Wunused:explicits", "-Wunused:privates",
-      "-Wconf:cat=unused-imports&src=routes/.*:s"),
+    scalacOptions := scalacOptions.value
+      .diff(Seq("-Wunused:all")) ++ Seq("-Wconf:src=routes/.*:s", "-Wconf:msg=Flag.*repeatedly:s"),
 
-    Test / scalacOptions ++= Seq("-Wunused:imports", "-Wunused:params",
-      "-Wunused:patvars", "-Wunused:implicits", "-Wunused:explicits", "-Wunused:privates"),
+    Test / scalacOptions ++= Seq(
```

Comment:
are all these scalacOptions needed here? we are removing most of them from the non-tests.

Reply:
For the testing options we still keep them, as they have been kept in other repos as well shown [here](https://github.com/hmrc/customs-duty-deferment-frontend/blob/e097b8bd957e44dcd853cb3c62515a64b64715b4/build.sbt#L25) and [here](https://github.com/hmrc/customs-cash-account-frontend/blob/879f01c789f65cfaf60c5ddf09cf12798279d743/build.sbt#L53), but yes we have removed them from non-testing options